core/src/main/java/org/mobilitydata/gtfsvalidator/validator/DefaultFieldValidator.java

```diff
@@ -108,12 +101,10 @@ public boolean validatePhoneNumber(
               cellContext.csvRowNumber(),
               cellContext.fieldName(),
               phoneNumber));
-      return false;
     }
-    return true;
   }
 
-  static boolean hasOnlyPrintableAscii(String s) {
+  public static boolean hasOnlyPrintableAscii(String s) {
```

Comment:
Why does this need to have it's visibility increased?

Reply:
Let's keep it package-private. That was unrelated.